### ISSUE TITLE
Add "Mean of 3" for FMC?

### ISSUE THREAD

**Issue by lgarron** [thewca/wca-regulations#81]
Proposal by Sébastien:

> The issue is very simple: Noone can deny the reasoning why we use averages to determine the winner for so many speedsolving events also applies for FMC. While using an average format for FMC is absolutely logical, the only issue that prevents us from doing this is the time needed. However, the number of competitions hosting more than one FMC attempt has strongly increased during the last years and I am convinced, that once the format is available, more organisers will decide to hold 3 attempts of FMC to provide the chance for the competitors to get a Mean of 3 result. 
> 
> After talking to many people over a long time, including the current board members, and after starting a discussion on Speedsolving to discuss this topic (http://www.speedsolving.com/forum/showthread.php?34216-WCA-Regulations-2014-Mean-of-N-for-FMC), I see that most people fully agree that Mean of 3 should be a possible format for FMC. Thus I am fully convinced that it is time to allow "Mean of 3" as official for FMC. The rest will be up to delegates and organisers.
> 
> In formal I ask you to consider the following for 2014's draft:
> - Move "Rubik's Cube: Fewest Moves" from 9b3 to 9b2. 
> 
> As an addition it bothers me (and others), that Best of 3 is possible for events that can be held using Mean of 3. Hence I also ask you to update 9b2a to:
> 
> 9b2a) Competition formats for these events are: "Best of X" (where X is 1 or 2) and "Mean of 3".

**Comment by sarahstrong314** [comment]
I don't see anything negative about changing FMC to the Mean of 3 format, since not too much would actually change in a competition setting. I definitely like the idea that competitions can still hold 1 or 2 attempts, where the best attempt would win just like before. It'd be very nice to have another set of rankings that display the means, which eliminate a lot of the bias in lucky solutions, and show who the top FM solvers really are. The only thing I think would be a problem would be if different delegates and organizers from different competitions were inconsistent about whether the mean or the best attempt wins when three attempts are held. I think it should be standard that mean has higher precedence when three attempts are held.

**Comment by SAuroux** [comment]
Hi Sarah, thanks for your comment. Your concern is addressed with the second part of my proposal, which is disallowing "Best of 3" for "Mean of 3" events.

**Comment by lgarron** [comment]
Sébastien, could you write a proposal for this? Something like [this](https://github.com/cubing/wca-documents/issues/49), including:
- Link to proposed change: https://github.com/cubing/wca-documents/compare/mean-of-3
- pros
- cons
- community concerns

It can be short, but I would prefer if you were the one to do it.

(Also, if you still want to change 9ba2, please make a [new issue](https://github.com/cubing/wca-documents/issues/new).)

**Comment by SAuroux** [comment]
Sure, I will find the time to do it this evening or tomorrow.

2013/12/8 Lucas Garron notifications@github.com

**Comment by SAuroux** [comment]
# Proposal (December 10, 2013)

**Adding "Mean of 3" for FMC.**
View exact proposed changes: https://github.com/cubing/wca-documents/compare/mean-of-3

## Pros
- the reasoning why averages are used to determine the winner for most WCA events - lucky solves - also applies for FMC. Many people even argue, that lucky solves have a bigger impact on FMC than on other speedsolving events, as competitors have many possibilities instead of just one within the 60 minutes to find very lucky solutions, that even the best in the world cannot beat without getting lucky. A "Mean of 3" would relax this issue by a lot. 
- the community generally agrees that "Mean of 3" should be a possible format for FMC (see http://www.speedsolving.com/forum/showthread.php?34216-WCA-Regulations-2014-Mean-of-N-for-FMC)
- the number of competitions hosting more than one FMC attempt has strongly increased during the last years. Once "Mean of 3" is available, even more organisers will decide to hold 3 attempts to provide the chance for competitors to obtain a "Mean of 3" result. The fact, that FMC requires very few ressources and can thus probably be held in parallel more easily as other events will benefit this.
- in contrast to some people's assumptions, organisers will of course still have the possibility to host FMC with a "Best of X" format, just like this is possible for all other events listed in 9b2.
- while some people argue that many people might DNF a "Mean of 3" in FMC, this argument doesn't bear closer examination. FMC even seems to be the easiest event to get a valid result for, as a competitor has a fixed time frame (60 minutes) in which he can find a result, write it down and check it many times before submitting it.

## Cons
- because of the necessary time, "Mean of 3" for FMC will certainly not be offered as frequently as the prefered formats for other WCA events.
- some competitors will have to change their tactic FMC: instead of "great solution or DNF", people will have to go for "the best possible result within 60 minutes".
- [Added by Lucas on GitHub] There may be more pressure on organizers from competitors who would like to get a mean of 3 once the ranking exists (e.g. to be competitive, to boost statistics, or just to have gotten every possible single/average/mean).

**Comment by lgarron** [comment]
Some comments:
- There is no poll in the linked speedsolving.com thread, so I can't tell if it's a majority. (Scrolling down the first 40 posts, I mostly see disagreement?) While we have to be careful about using the (possibly biased) posts of such a thread, we can certainly say there's some support, or that people are neutral.
- In the US, one main concern is that many organizers at US Nationals like to do FMC (sometimes just to guarantee an hour without other responsibilities). This greatly hinders doing anything in parallel.

**Comment by lgarron** [comment]
For the record, this is GitHub issue
#81https://github.com/cubing/wca-documents/issues/81.
From what I wrote there:

In the US, one main concern is that many organizers at US Nationals like to
do FMC (sometimes just to guarantee an hour without other
responsibilities). This greatly hinders doing anything in parallel.

Realistically, I believe community support is mostly like Skewb: a
significant number or people would like it, a handful have their reasons
against it, and most people would be okay with it.

»Lucas Garron

**Comment by KitClement** [comment]
Reposting here what I said in the speedsolving forums:

"I like the idea for championships, but not in general. I know that it has been mentioned that the FMC Mean rankings "wouldn't mean anything," but I still don't like having formats appear on a WCA profile that are so rare, as a significant amount of competitors simply wouldn't have the resources to attend a competition where an FMC Mean of 3 is offered. 

Since awards can be given at the organizer's discretion, why not simply award the winners based on Mean of 3? I know the WCA competition results page will not reflect the actual award winners, but I think at a major championship with large prize money at stake, this will be the competitors' main goal rather than appearing on the WCA page."

If we want to make FMC a better event in the sense that it should remove luck and still be a time-friendly event, then I propose this.  (Note that I have never attempted FMC myself, so I don't know how feasible this might be)
- Competitor is given all 3 scrambles simultanously
- The competitor has 90 minutes (this time can be changed) to come up with a solution for all 3 scrambles
- The attempt will be listed as 3 separate attempts, with each of the 3 scrambles representing single times, and the average format is mean of 3.  (maybe don't recognize single, as competitors can devote all time to one scramble to get WR single)
- Competitors will be ranked by best mean, and ties are determined by best single.

Clearly, this format is far different from the current FMC, and would have to be listed as a separate event.  This would similar to what we did with MBLD in 08, but we may not necessarily need to remove the current FMC format.  But this at least makes this feasible to be run at any competition rather than only major competitions.  I don't expect this to be a well-received proposal, but maybe it can at least spark ideas for a better way of looking at this event.

**Comment by SAuroux** [comment]
- The ranking in our Misc section does already represent surely more than nothing. When the format is available, there will be 5-10 times more results until the end of 2014, trust me. The people that really care will surely find their chance to attend such a competition or maybe even just be encouraged to organise one.
- Your thoughts about the awards are too optimistic in my opinion. If it is not official, most people will oppose this and it is simply not going to happen.
- Without going much into detail: thanks for making me see approximately the 100th alternate proposal, which includes 1. a weird new format, 2. requires abolishing 10 years of FMC results, 3. coming from a person that has no or almost no experience with FMC. Yours it not even one of the better ones. I'm really getting tired of this, sorry. Oh wait, 2. is actually not true for your proposal, as you propose to add a 2nd FMC WCA event taking 90 minutes. That makes of course much more sense than adding Mean of 3 as an option and everyone will sure be getting a resultes for this! I mean, come on, you can't be serious...
- **Mean of 3 is the only solution to the problems we currently have with FMC which is consistent with our regulations!** I'm still very confident that it will work way better in practise as those with doubts are expecting. I have tried to support this with many statistics on Speedsolving.com so far, but we can only really now how it will work if we give it a try.
- To accomodate the format, we could indeed reduce the time provided per attempt, but that shouldn't be a too significant change, to not lose the results that we already have. 45 minutes for example could be reasonable. But that's a different topic to discuss, and I'd rather postpone this to next year and bring it up if and only if we see that Mean of 3 works too bad in practise with 60 minutes per attempt.

**Comment by KitClement** [comment]
I had not realized that people have proposed alternate formats for FMC, so I'm sorry that mine came off as bizarre.  I probably wouldn't have brought it up had I known many others have proposed something similar to you.  I knew mine was crap, I just wanted to introduce the idea of a new format, which apparently was not necessary at all.

I do like the option of providing Mean of 3 as an event format, but I just don't like the limited accessibility of the event.  It may make more sense for the two day competitions in Europe, but it really doesn't make sense in the USA where competitions are only one day, outside of Nationals.  

As much as I like it as an organizer to make a mean of 3 attempt be 45 minutes per attempt, I am not a fan of changing a key part of the format and calling it the same event.  Maybe we could still allow 60 minutes per attempt in a "Best of X" format, but require 45 minutes for "Mean of 3"?

**Comment by SAuroux** [comment]
Having different rules for solves in a Best of format and solves in a Mean format? That's just another weird proposal without any consistent background...

No, I'm just keeping up my proposal. And the community's majority seems to be in favour of it. I'm pretty confident that it is going to work, even in the US. Why shouldn't it? Why wouldn''t someone want to organise a competition hosting Mean of 3 if he cares enough? If you still doubt this, then we will simply have to wait and find it out. :)

**Comment by KitClement** [comment]
Yeah, it wasn't a very well thought out idea.  :P

I have a hard time seeing how you could run FMC Mean of 3 in a one-day competition without sacrificing a significant amount of time, and a very select few would rather see this FMC format than the events that would be lost.  But I agree, no harm in giving it a shot, just wanted to let my concerns be heard.
